281 doc gestionale esterno verb est 2026 07 06 (#290)
Synergo-Unit-UniPD/Documentary-Repo: docs/PB/doc_gestionali/verb_esterni/verb_est_2026_07_06/contenuto.tex

\begin{itemize}
	\item \textbf{Azienda Coinvolta:} Zucchetti\textsubscript{G}
	\item \textbf{Mezzo di Comunicazione:} Google Meet
	\item \textbf{Data:} 2026-07-06
	\item \textbf{Orario di inizio:} 15:00
	\item \textbf{Orario di fine:} 16:50
\end{itemize}

Il presente verbale documenta l’incontro, svoltosi tramite Google Meet, 
tra il gruppo Synergo Unit e il referente dell’azienda Zucchetti. 

> Durante la riunione sono stati presentati e discussi il PoC(Proof of Concept), i problemi dell'infastruttura, i requisiti non funzionali e la disponibilità dell'azienda Zucchetti durante il periodo estivo.

Reviewer: ```suggestion
Durante la riunione sono stati presentati e discussi il PoC (Proof of Concept), i criticità dell'infastruttura LLM, alcuni requisiti non funzionali e gli aspetti organizzativi relativi alla disponibilità dell'azienda durante il periodo estivo.
```

Il gruppo ha richiesto chiarimenti sui seguenti aspetti:

\begin{enumerate}
	\item Proof of Concept

> 	\item Problemi dell'infastruttura

Reviewer: ```suggestion
	\item \item Criticità dell'infrastruttura LLM
```

	\item Requisiti non funzionali
	\item Disponibilità dell'azienda durante il periodo estivo
\end{enumerate}

% --- 3. RESOCONTO ---
\section{Resoconto}

\subsection{Proof of Concept}

Il Proof of Concept è stato visionato e approvato.

\subsection{Problemi dell'infastruttura}

> Per quanto riguarda l’infrastruttura attuale, è stato suggerito di utilizzare inizialmente il modello *gemma4:12b*. Eventualmente, potranno essere ampliati i nostri permessi per consentire l’utilizzo del modello *Qwen3.6-35B-A3B*, ritenuto più stabile e con prestazioni superiori. È stato inoltre raccomandato di evitare chiamate a cascata tra modelli, in quanto l’infrastruttura prevede il ricaricamento del modello ad ogni richiesta, con conseguente perdita di efficienza.

Reviewer: ```suggestion
Per quanto riguarda l’infrastruttura attuale, è stato suggerito di utilizzare inizialmente il modello  \texttt{gemma4:12b}. Qualora necessario, potranno essere ampliati i permessi di accesso per consentire l’utilizzo del modello \texttt{Qwen3.6-35B-A3B}, ritenuto più stabile e con prestazioni superiori. È stato inoltre raccomandato di evitare chiamate a cascata tra modelli, in quanto l’infrastruttura prevede il ricaricamento del modello ad ogni richiesta, con conseguente perdita di efficienza.
```

> Il Proof of Concept è stato visionato e approvato.

Reviewer: ```suggestion
Il gruppo ha presentato il funzionamento del Proof of Concept, illustrandone l'architettura e le principali funzionalità implementate. Al termine della dimostrazione il referente ha espresso un riscontro positivo, ritenendo il PoC adeguato agli obiettivi previsti.
```

\subsection{Requisiti non funzionali}
I requisiti non funzionali dovranno essere ampliati e resi maggiormente misurabili. In particolare:
\begin{itemize}
    \item \textbf{Compatibilità browser:} è necessario definire i browser ufficialmente supportati e le eventuali versioni minime. Browser come Edge, Safari e Chrome condividono la stessa base tecnologica e non dovrebbero presentare criticità rilevanti; al contrario, Firefox utilizza un motore differente e richiede test specifici, così come Safari su macOS.

>     \item \textbf{Prestazioni AI:} occorre chiarire quale tempo di attesa sia considerato accettabile per la generazione di una proposta AI. In precedenza era stato indicato un tempo minimo di 2 secondi; si è deciso tuttavia di non fissare un limite massimo, introducendo invece un meccanismo di interruzione manuale che consenta all’utente di interrompere l’elaborazione.

Reviewer: ```suggestion
        \item \textbf{Prestazioni AI:} occorre chiarire quale tempo di attesa sia considerato accettabile per la generazione di una proposta AI. È stato confermato il tempo minimo di visualizzazione dell'indicatore di caricamento pari a circa 2 secondi, mentre si è deciso di non imporre un tempo massimo di elaborazione, introducendo invece un meccanismo di interruzione manuale che consenta all'utente di arrestare la generazione della proposta in qualsiasi momento.
```

>     \item \textbf{Persistenza opzionale:} è stato chiarito che il requisito opzionale relativo al database comprende funzionalità complete di autenticazione e gestione delle note (visualizzazione elenco, apertura, aggiornamento ed eliminazione), e non la sola memorizzazione. L'eliminazione delle note è considerata meno rilevante in quanto rappresenta una funzionalità standard che non introduce elementi di particolare innovazione.

Reviewer: ```suggestion
    \item \textbf{Persistenza opzionale:} è stato chiarito che il requisito opzionale relativo al database comprende funzionalità complete di autenticazione e gestione delle note (visualizzazione elenco, apertura, aggiornamento ed eliminazione), e non la sola memorizzazione. È stato inoltre evidenziato che tali funzionalità, pur essendo necessarie per una gestione completa della persistenza, rappresentano caratteristiche standard e non costituiscono gli elementi di maggiore innovazione del progetto, che rimangono invece le funzionalità di supporto alla scrittura basate su IA.
```

\end{itemize}

\subsection{Disponibilità dell'azienda durante il periodo estivo}

> Per quanto riguarda il periodo estivo, considerata la presenza di ferie, il docente ha suggerito di valutare la possibilità di individuare preventivamente degli slot di comunicazione, al fine di evitare ritardi dovuti a difficoltà di coordinamento. È stato inoltre comunicato che l’azienda resterà chiusa nelle due settimane centrali di agosto e che il referente sarà assente per l’intero mese. Si raccomanda pertanto di concentrare le comunicazioni entro la fine di luglio e l’inizio di settembre, mantenendo come eventuali finestre operative la prima e l’ultima settimana di agosto in caso di necessità.

Reviewer: ```suggestion
Per quanto riguarda il periodo estivo, il gruppo ha verificato con il referente la disponibilità dell'azienda, in linea con quanto pianificato internamente per individuare preventivamente finestre temporali dedicate alle comunicazioni e ridurre il rischio di ritardi dovuti al periodo di ferie. È stato comunicato che l'azienda resterà chiusa nelle due settimane centrali di agosto e che il referente sarà assente per l'intero mese. Di conseguenza, è stato concordato di concentrare le comunicazioni entro la fine di luglio e a partire dall'inizio di settembre, mantenendo come eventuali finestre operative la prima e l'ultima settimana di agosto in caso di necessità.
```

	\bottomrule
	\endlastfoot

	\textbf{VE-7.1}
		& Proof of Concept approvato \\[4pt]

	\textbf{VE-7.2}

> 		& Scelte riguardo i problemi dell'infastruttura \\[4pt]

Reviewer: ```suggestion
		& Adozione del modello \texttt{gemma4:12b} come riferimento iniziale e valutazione di un futuro utilizzo del modello \texttt{Qwen3.6-35B-A3B}. \\[4pt]
```

	\textbf{VE-7.3}

> 		& Scelte riguardo i requisiti funzionali \\[4pt]

Reviewer: ```suggestion
		& Revisione dei requisiti non funzionali relativi a compatibilità, prestazioni e persistenza. \\[4pt]
```

	\bottomrule
	\endlastfoot

> 

Reviewer: ```suggestion

\ghissue{282}
		& VE-7.3
		& Correzione e integrazione AdR Macroarea 1 e 3
		& Armando Moda Scarati
		& 2026-07-14 \\[4pt]
		
\ghissue{283}
		& VE-7.3
		& Correzione e integrazione AdR Macroarea 2
		& Filippo Idiometri
		& 2026-07-14 \\[4pt]
		
```
